src/providers/push/pubnub/pubnub.provider.ts

```diff
@@ -36,15 +41,34 @@ export class PubNubProvider implements Provider<PubNubNotification> {
               title: message.subject,
               description: message.body,
               // eslint-disable-next-line @typescript-eslint/naming-convention
-              pn_gcm: {
-                data: Object.assign(
-                  {
-                    title: message.subject,
-                    description: message.body,
-                  },
-                  message.options,
-                ),
-              },
+              pn_gcm:
```

Comment:
set the previous value as default, so in case no config is provided, the older config is used.